subprojects/smoke-test/src/smokeTest/resources/org/gradle/smoketests/cache-init-scripts/build.gradle

```diff
@@ -1,3 +1 @@
-apply from: "overlapping-task-outputs-stats-init.gradle"
```

Comment:
This file can go away completely as well.

Reply:
Ah, you're right. Actually, the whole directory can come out.